.travis.yml

```diff
@@ -4,3 +4,5 @@ python:
 
 script:
   - bash ./run_travis_tests.sh
+
+sudo: false
```

Comment:
This is to switch to the Travis containers and remove the warning about using deprecated infra.